oauth2client/file.py

```diff
@@ -20,17 +20,28 @@
 
 import os
 import threading
+import warnings
 
 from oauth2client import client
 
 
 __author__ = 'jcgregorio@google.com (Joe Gregorio)'
 
+_SYM_LINK_MESSAGE = 'File: {0}: Is a symbolic link.'
+
+_IS_DIR_MESSAGE = '{0}: Is a directory'
+
+_MISSING_FILE_MESSAGE = 'Cannot access {0}: No such file or directory'
+
 
 class CredentialsFileSymbolicLinkError(Exception):
     """Credentials files must not be symbolic links."""
 
 
+class CredentialsFileDirectoryError(Exception):
```

Comment:
Subclass `IOError`? Why not simply use `IOError`?

The circumstance at play here is a programming defect, right? If this exception (or these exceptions) are ever raised in the wild, no user of `oauth2client` is going to modify their code to handle the exception, right? They're just going to say "oh, my code has a bug" and they're going to fix it?

Why not just use `IOError` to communicate the problem? It's the exception's message and not its class that seems to be doing all the communication anyway?